es6/jsxcad-ui-app.js

```diff
@@ -2860,6 +2860,36 @@ class EditNote extends ReactDOM$3.Component {
   }
 }
 
+class ErrorNote extends ReactDOM$3.Component {
```

Comment:
What do these look like in practice?

Reply:
Red text like "TypeError: Box(...).ez(...).clean(...).And(...).clip(...).flub is not a function"